### PR TITLE
chore(dev): :heavy_plus_sign: add nodemon for better DX and remove warnings

### PR DESCRIPTION
**Pull request by iamaamir**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -105,12 +90,16 @@ app.use(
...
+    saveUninitialized: true
   })
 );
 app.use(flash());
 app.use(busboy());
-app.use(bodyparser.urlencoded());
-
+app.use(express.json());
+app.use(express.urlencoded({
+  extended: true
```
Comment (serverwentdown) [inline index.js:101]: I feel like `extended` is not necessary as it isn't used yet. We can enable it when it is needed.

File: index.js [hunk 2]
```diff
@@ -105,12 +90,16 @@ app.use(
...
     secret: process.env.SESSION_KEY || "meowmeow",
+    resave: true,
+    saveUninitialized: true
   })
 );
 app.use(flash());
 app.use(busboy());
-app.use(bodyparser.urlencoded());
-
+app.use(express.json());
```
Comment (serverwentdown) [inline index.js:99]: Why is this needed?

File: index.js [hunk 3]
```diff
@@ -105,12 +90,16 @@ app.use(
 app.use(
   session({
     secret: process.env.SESSION_KEY || "meowmeow",
+    resave: true,
+    saveUninitialized: true
```
Comment (serverwentdown) [inline index.js:94]:
```suggestion
    resave: false,
    saveUninitialized: false
```

Avoids [saving cookies when not needed](https://github.com/expressjs/session#resave)

**Review by serverwentdown — CHANGES_REQUESTED** [review]
Awesome! Thanks for contributing

**Comment by iamaamir** [comment]
closing in favor of #22

**Pull request closed without merging**